src/components/ebay-skeleton-loader/index.marko

```diff
@@ -0,0 +1,48 @@
+import { processHtmlAttributes } from "../../common/html-attributes";
+static var ignoredAttributes = [
+    "a11yText",
+    "class",
+    "size",
+    "type",
+    "multiline",
+    "count",
+    "columns",
+    "isVisible",
+];
+static var validTypes = ["avatar", "button", "text", "form", "image", "tile"];
+static var validSizes = ["large", "small"];
+$ var size = validSizes.includes(input.size) ? input.size : "default";
+$ var type = validTypes.includes(input.type) ? input.type : "button";
+$ var typeClass = type === "button" ? "skeleton--btn" : `skeleton--${type}`;
+$ var skeletonClass = (
+    size !== "default"
+        ? `${typeClass}-${size}`
+        : type === "text"
+        ? `${typeClass}-small`
+        : typeClass
+);
+
+<if(typeof input.isVisible === "undefined" ? true : input.isVisible)>
```

Comment:
Using `if` here means that we need to use JavaScript to create and destroy DOM nodes, which means that layout shift is likely and also a lot of work is being done each time a skeleton is shown or hidden. I think it would probably be preferable to toggle `style="visibility:hidden"` on the div instead.

Reply:
The idea for using `isVisible` is to set the flag to false to remove skeleton from dom and show content below. This is useful technique for hiding sibling skeleton. Preferably teams use other techniques like if-else or await placeholder than this approach. If we use `visibility:hidden`, won't the empty layout still visible beside the regular content if not used content swap approach?

Reply:
Yes we should keep it like this. Visibility hidden wont work
At the same time the tricky part of layout shift isnt hiding elements but ensuring that everything is the same size

Reply:
Wait I still disagree with this at a fundamental level. Right now all `isVisible` does is control a top-level if statement, so really we should just remove this entirely and have teams use
```marko
<if(isVisible)>
  <ebay-skeleton-loader />
</if>
```

Reply:
yeah, for now I removed the wrapper condition and `isVisible`. Instead let's use documentation to explain the right techniques.